hash/hp9825a_rom.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0
+
+HP9825A option ROMs
+Compiled by F.Ulivi with data from http://www.hpmuseum.net/
+-->
+
+<softwarelist name="hp9825a_rom" description="HP 9825A Option ROMs">
```

Comment:
At least some of these are duplicates of items in hp9825b_rom.xml – please turn these into a single software list with compatibility filters.